plugin.video.srfplaytv/addon.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.srfplaytv" name="SRF Play TV" version="0.9.2" provider-name="Alexander Seiler">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+    <import addon="script.module.simplecache" version="1.0.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="addon.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">SRF Play television video library</summary>
+    <summary lang="de">SRF Play Videobibliothek</summary>
+    <description lang="en_GB">SRF Play TV allows you to watch the content from the video library of Swiss Radio and Television (SRF).</description>
+    <description lang="de">Mit SRF Play TV können Sie den Inhalt der SRF Play Videobibliothek des Schweizer Fernsehens anschauen.</description>
```

Comment:
de_DE

Reply:
Since the content is from a Swiss provider, should I use "de_CH" instead of "de_DE"? I assume de_DE refers to the German language spoken in Germany. Is de_CH a valid language label too?

Reply:
I suggested de_de since i am not completetly sure de_ch is valid (could not find it in the estuary language folder). I would say to give it a go and ser if it works (after a restart). If not use de_de :)